releasenotes/notes/0.15/remove-deprecated-pulse-commands-18381c59264090dd.yaml

```diff
@@ -0,0 +1,41 @@
+---
+upgrade:
+  - |
+    The ``qiskit.pulse.commands`` module containing ``Commands`` classes
```

Comment:
```suggestion
    The :mod:`qiskit.pulse.commands` module containing ``Commands`` classes
```

Reply:
I didn't do this because `qiskit.pulse.commands` has been removed so there is nothing to link to by using `:mod:`